### PR TITLE
Improve code

### PR DESCRIPTION
**Pull request by Dev-Shaz**
(#vec1-vec2) is better to use then GetDistanceBetweenCoords.

GetPlayerPed(-1) is old and PlayerPedId()  is better to use.

### CODE REVIEW

**Comment by Rubylium** [comment]
Looks pretty good to me expect on indentation issue in utils.Lua -> GetClosestPlayer but could be GitHub displaying weirdly the code :)

Did all those change got tested ?

**Comment by Dev-Shaz** [comment]
I did not test them as they were only small/basic changes

**Comment by Rubylium** [comment]
Mmh, I prefer if it’s tested, just in case ^^
can’t really test it myself rn so I lot you do that and I will approuve the PR !

File: [gamemodes]/driftv/client/utils.lua [hunk 1]
```diff
@@ -480,7 +480,7 @@ function GetClosestPlayer()
 		if GetPlayerPed(v) ~= pPed then
 			local oPed = GetPlayerPed(v)
 			local oCoords = GetEntityCoords(oPed)
-			local dst = GetDistanceBetweenCoords(oCoords, coords, true)
+            local dst = #(oCoords - coords)
```
Comment (rdelian) [inline [gamemodes]/driftv/client/utils.lua:483]: Line using spaces when _general_ indentation for the file it's done using tabs

**Pull request closed without merging**